doc/stages/readers.slpk.rst

```diff
@@ -0,0 +1,43 @@
+.. _readers.slpk:
+
+readers.slpk
+===========
+
+`Scene Layer Packages (SLPK)`_ is a specification created by Esri as a format for their 3D Scene Layer and scene services. SLPK is a format that allows you to package all the necessary I3S files together and store them locally rather than find information through REST.
+
+Example
+--------------------------------------------------------------------------------
+This example will unarchive the slpk file, store it in a temp directory, and traverse it. The data will be reprojected to another spatial reference, and output to a las file. This is done through PDAL's command line interface or through the pipeline.
+
+.. code-block:: json
+    {
+        "pipeline":[
+            {
+                "type": "readers.slpk",
+                "filename": "PDAL/test/data/i3s/SMALL_AUTZEN_LAS_All.slpk"
+                "bounds": "([-123.075542,-123.06196],[44.049719,44.06278])"
```

Comment:
Same as I3S doc comment.  Can these two docs be combined or one refer to the other?  Seems error-prone to have copy-pasted docs.